sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc_client.php

```diff
@@ -1027,7 +1069,25 @@ function build_interface($lcd) {
 					$lcd_cmds[] = "widget_add {$name} title_wdgt string";
 					$lcd_cmds[] = "widget_add {$name} text_wdgt scroller";
 					break;
-				case "scr_traffic":
+					case "scr_ntp":
+//						$lcd_cmds[] = "screen_add $name";
+//						$lcd_cmds[] = "screen_set $name heartbeat off";
+//						$lcd_cmds[] = "screen_set $name name $name";
+//						$lcd_cmds[] = "screen_set $name duration $refresh_frequency";
+//						$lcd_cmds[] = "widget_add $name title_wdgt string";
+//						$lcd_cmds[] = "widget_add $name text_wdgt scroller";
+//						$lcd_cmds[] = "widget_set $name title_wdgt 1 1 \"+ NTP Source\"";
```

Comment:
No need to add these lines since they are commented out

Reply:
removed